resources/i18n/it_IT/cura.po

```diff
@@ -119,7 +118,7 @@ msgstr "&Salva progetto..."
 
 msgctxt "@title:menu menubar:file Don't translate 'Universal Cura Project'"
 msgid "&Save Universal Cura Project..."
-msgstr ""
+msgstr "e salvare il progetto universale Cura..."
```

Comment:
"e" is not needed and I think in other translation is used "Salva"

Reply:
So I assume in the end it is "Salva il progetto universale Cura..." ?
Also "Universal Cura Project" is something we didn't want translated, so I will restore it.

Reply:
Yep at the end "Salva Universal Cura Project"